scripts/prettify-changed.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+get_changed() {
+    # Get a list of changed and committed files with appropriate extensions
+    git diff master...HEAD --name-only --diff-filter=d | grep -E "\.(html|css|scss|less)$"
+}
+
+prettify() {
+    FILES=$1
+    echo "Formatting files changed on this branch:"
+    ./node_modules/.bin/prettier --write "$FILES"
+    echo
+}
+
+CHANGED_FILES=$(get_changed) || { echo "No changed files found to format." && exit 1; }
+prettify "$CHANGED_FILES" || { echo "Oops! Did you remember to run yarn install?" && exit 1; }
```

Comment:
Is it possible/expected that there will be errors that cause this "Oops! ..." output, but to which it is not a useful suggestion?

Reply:
Technically possible if Prettier hits a parse error while reading any of the files passed in, but since we're limiting by filetype this is usually okay -- the rare exception being knockout/embeddedJS type templates that use `<% %>` and do so outside of `<script>` tags. In those cases you'd see something like this from Prettier:
```console
[error] corehq/apps/cloudcare/templates/cloudcare/partials/pagination.html: SyntaxError: Opening tag "a" not terminated. (46:21)
```

I added that "Oops" message because the "no such file or directory" error when `./node_modules/.bin/prettier` isn't found is not particularly helpful.